### ISSUE TITLE
Document how gene interactions are modeled

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/GENO-ontology#25]
A good question from @kyook (see the slides in Dropbox for background)

```
I don’t understand where gene interactions are accounted for.  We separate genetic background - like in your example of the daf-2;fog-4 mutant, in this case, as displayed fog-2 would be considered a background mutation, one that needs to be there to assess the phenotype, but in slide 4 why is fgf8 homozygosity not considered the genetic background?  In each experiment, the phenotype needs to be assessed based on a control genotype. In this case, the control genotype would be fgf8 wouldn’t it?
```

**Comment by mbrush** [comment]
Yes, good catch.  The slides you looked at are likely an early iteration where we allowed for 'promiscuous' phenotype propagation.  Specifically, in cases where phenotypes were annotated to 'complex' genotypes (e.g. a double-mutant where two genes are variant), we propagated phenotypes down the genotype partonomy to each variant allele and affected gene.  As you point out, this does not account for the facts that: (1) only one of the affected genes is likely causative for the observed phenotypes, and (2) there may be genetic interactions between the variant genes that cause or modulate the phenotypes in the double mutant.  We are currently implementing heuristic-based approach to constrain such inappropriate phenotype propagation from genotypes to their component alleles and affected genes - similar to the rules applied by MGI as described in the paper [here](https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0054-4), and by ZFIN to generate the phenoGeneCleanData file [here](https://zfin.org/downloads/phenoGeneCleanData_fish.txt).  

The updated slide deck [here](https://github.com/monarch-initiative/GENO-ontology/blob/develop/docs/GENO_Overview%282016-06-12%29.pptx) is a bit more up to date, in describing these issues.  Hoping these slides and the accompanying notes are more clear - and we would welcome any further feedback or questions you have!
